Cargo.toml

```diff
@@ -12,4 +12,4 @@ Un planificador semanal de comidas.
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-chrono = "0.4.23"
+time = "0.3.17"
```

Comment:
REcuerda que en este objetivo no podéis usar ninguna dependencia externa; se trata de que toda la estructura de negocio que uséis sea vuestra y hagáis tests "de caja blanca" que examinen todos los caminos de código que *vosotros* hayáis creado.